lib/scenario_runner.py

```
@@ -1333,18 +1335,64 @@ def _setup_services(self):
+                        mount_option = '' # read-write by default. But will work only with allow list
+
+                        if vol_len == 3:
+                            if vol[2] == 'ro' or vol[2] == 'readonly':
+                                mount_option = ',readonly'
+                            else:
+                                raise ValueError(f"Service '{service_name}': We only allow readonly (ro) or no parameter (writeable) for volume mounts. Volume: {volume}")
+
+                        mount_string = f"{mount_src}{mount_option}"
+                        if mount_string in self._allowed_volume_mounts:
+                            if '/' not in mount_src: # volume case. should exist
+                                ps = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)